lib/kerberos.js

```diff
@@ -14,6 +14,15 @@ var Kerberos = function() {
 // delegation), specify the cache name here and it will be used for this 
 // exchange. The credentialsCache is optional.
 Kerberos.prototype.authGSSClientInit = function(uri, flags, credentialsCache, callback) {
+  return authClientInit.call(this, uri, flags, credentialsCache, callback);
+}
+
+// uses mechanism specific OID 
+Kerberos.prototype.authGSSClientInitDefault = function(uri, flags, credentialsCache, callback) {
+  return authClientInit.call(this, uri, flags, credentialsCache, callback, Kerberos.GSS_C_NO_OID);
```

Comment:
convention is that `callback` should always be the final parameter for a function, its fine to break the API here because this is private implementation - could you please swap the `oid` and `callback` parameters?